_scss/partials/_list-icons.scss

```diff
@@ -29,3 +30,9 @@ li.fix::before {
 li.bug::before {
   background: url(../i/icons/bug.svg);
 }
+
+li.flag::before {
+  background: url(../i/icons/flag.svg);
+  background-repeat: no-repeat;
```

Comment:
Just curious, why do you need a `no-repeat` here? The icon is supposed to be clipped by the width of the pseudo element, and should not repeat... 

If needed, please add the style to the general class (line 10)

Reply:
Yeah, for some reason I had to add the no repeat to this svg. I couldn't understand why since the sizing of it was the same as the other svgs. I'll add it to the general class.